images/miq-app/ose-assets/appliance-initialize.service

```diff
@@ -1,13 +1,13 @@
 [Unit]
 Description=Initialize Appliance Database
-After=evminit.service memcached.service
+After=evminit.service
 Before=evmserverd.service
-Requires=memcached.service
 [Service]
 TimeoutStartSec=10m
 Type=oneshot
+EnvironmentFile=/container.env.vars
```

Comment:
@fvillain I like this, now that we supply the container environment to /bin/appliance-initialize.sh , we should be able to get rid of ${APP_ROOT}/tmp/ose-env, it seems more elegant. The only reason that ose-env existed was to ensure systemd was importing the correct OSE environment variables before running the database preparation.